app/views/backdate/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+<% content_for :title, t("backdate.edit.title") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_govspeak(t("backdate.edit.intro_paragraph")) %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% legend = capture do %>
+      <span class="govuk-heading-s govuk-!-margin-bottom-0">
+        <%= t("backdate.edit.date") %>
+      </span>
+    <% end %>
+    <% date = @edition.backdated_to %>
+
+    <%= form_tag backdate_path(@edition.document) do %>
+      <%= render "govuk_publishing_components/components/date_input", {
+        legend_text: legend,
+        hint: t("backdate.edit.date_hint_text"),
+        name: "backdate",
+        id: "date",
+        error_items: @issues&.items_for(:backdate),
+        items: [
+          {
+            name: "day",
+            width: 2,
+            value: params.dig(:backdate, :day) || date&.day,
+          },
+          {
+            name: "month",
+            width: 2,
+            value: params.dig(:backdate, :month) || date&.month,
+          },
+          {
+            name: "year",
+            width: 4,
+            value: params.dig(:backdate, :year) || date&.year,
+          }
+        ]
+      } %>
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Save",
+        margin_bottom: true
+      } %>,
```

Comment:
Stray comma here!

<img width="172" alt="Screen Shot 2019-06-19 at 14 37 33" src="https://user-images.githubusercontent.com/9029009/59770352-cb503f00-929f-11e9-9753-dafc5f2f3efe.png">

Reply:
I had spotted this before and meant to fix it, slipped my mind!